TebLocalPlannerROS:
  odom_topic: odom
  map_frame: map
  teb_autosize: True
  dt_ref: 0.3
  dt_hysteresis: 0.1
  global_plan_overwrite_orientation: True
  max_vel_x: 0.2
  max_vel_theta: 0.5
  acc_lim_theta: 2.0
  acc_lim_x: 2.0
  acc_lim_y: 0.0
  min_turning_radius: 0.0
  footprint_model:
    type: "circular"
    radius: 0.153
  robot_radius: 0.2
  xy_goal_tolerance: 0.2
  yaw_goal_tolerance: 0.1
  obstacle_avoidance: True
  min_obstacle_dist: 0.2
